lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_client/last_updated.rb

```diff
@@ -0,0 +1,28 @@
+module CarinForBlueButtonTestKit
+  class CARIN4BBLastUpdatedTest < Inferno::Test
+    title 'Uses meta.lastUpdated value appropriately'
+    description <<~DESCRIPTION
+      The Health IT Module must use the meta.lastUpdated value to determine if the associated data is
+      accurate as of the date of service or as of the current date.
+    DESCRIPTION
+    id :c4bb_last_updated
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@90', 'hl7.fhir.us.carin-bb_2.0.0@180',
+                          'hl7.fhir.us.carin-bb_2.0.0@188'
```

Comment:
Should we still include 188 even though that requirement is out of scope? Or is the out of scope requirements list out of date?

Reply:
made in scope